pkg/GAPJulia/JuliaInterface/read.g

```diff
@@ -0,0 +1,30 @@
+#
+# JuliaInterface: Test interface to julia
+#
+# Reading the implementation part of the package.
+#
+ReadPackage( "JuliaInterface", "gap/JuliaInterface.gi");
+ReadPackage( "JuliaInterface", "gap/BindCFunction.gi" );
+ReadPackage( "JuliaInterface", "gap/generated_path.gi" );
+
+dirs_gap_jl := Directory( _JULIAINTERFACE_JULIA_MODULE_SOURCES );
```

Comment:
I am aware of this, and the reason why I did all of this is different: At this point of the module/package loading stage, I simply cannot translate the Julia string back to GAP. So we have to choose one of those evils here:

1. Have a C-funtion that translates a Julia string to a GAP string
2. Have a C-function that does what you described above
3. Use the generated file with the path

If you really want to use your own version of JuliaInterface, then with the current setup you would have to not only copy over the `generated_path.gi`, but also change the `deps.jl` (because otherwise the GAP called by Julia will anyway try to load their own version of GAP). So What I would suggest is actually symlinking the `GAPJulia` folder into your local GAP pkg folder. I am not sure what this means for using `git`, though, because the `.git` folder on top of the symlink might be wrong.

Reply:
I dislike the `generated_path.gi` for various reasons; one is that it makes it harder to relocate the files. Which matters if one wants to package these things into a binary installer for users.

So I'd really prefer approach 1 or 2. I think 1 would be better, as it requires less and simpler C code, and might be handy for other things, too.

Reply:
OK, but I just see that `dirs_gap_jl` is not used at all anymore, so the even simpler solution is to remove all of this :)

Reply:
Yep, I already removed the `generated_path.gi` completely.